src/scss/layout/_main.scss

```diff
@@ -22,7 +22,7 @@
     grid-column: full;
   }
 
-  @media (min-width: 45ch) {
+  @media (width >= 45ch) {
```

Comment:
Similar, this is maybe an ok change - but it reduces backwards compat. Is a linter doing it, or is there a reason we want to?

Reply:
Yeah, this seems to be automatically changed by Stylelint: https://stylelint.io/user-guide/rules/media-feature-range-notation/